File: A.A.-2015-2016/Programmazione/20160108/c++/persona.cpp
#include "persona.h"
#include <stdio.h>

void persona::stampa()
{
	printf("Nome: %32s\n", this->nome);
	printf("Cognome: %32s\n", this->cognome);
	printf("Telefono: %32d\n", this->telefono);
}

persona::persona(const char* n, const char*c, int nt)	

{
	this->nome=n;
	this->cognome=c;
	this->telefono=nt;
}

File: A.A.-2015-2016/Programmazione/20160205/Java/anagrafica/Persona.java
public class Persona{
	public String nome, cognome;
	public int numTelefono;
	Persona(String n, String c, int num)
	{
		this.nome = n;
		this.cognome = c;
		this.numTelefono = num;
	}
	public void stampa()
	{
		System.out.println("Questo è il nome: " + nome + "\nQuesto è il cognome: " + cognome + "\nE puoi chiamarlo qui: " + numTelefono);
	}
}
